static/js/emoji_picker.js

```diff
@@ -19,6 +19,22 @@ var search_results = [];
 var section_head_offsets = [];
 
 function get_emoji_categories() {
```

Comment:
Actually it doesn't not return all the categories right now but the original idea behind this function is that it will return all the categories. I am having another PR which will make it return all the categories.